src/components/Footer/Footer.js

```diff
@@ -28,17 +35,17 @@ const Footer = () => (
         <h6 className={styles['footer__liens__title']}>Liens</h6>
         <ul className={styles['footer__list']}>
           <li className={styles['footer__list__other__row']}>
-            <Link to='/#' className={styles['footer__list__link']}>
+            <Link to={menu[1].path} className={styles['footer__list__link']}>
               À propos
             </Link>
           </li>
           <li className={styles['footer__list__other__row']}>
-            <Link to='/#' className={styles['footer__list__link']}>
+            <Link to={menu[0].path} className={styles['footer__list__link']}>
               Blog
             </Link>
           </li>
           <li className={styles['footer__list__other__row']}>
-            <Link to='/#' className={styles['footer__list__link']}>
+            <Link to={menu[2].path} className={styles['footer__list__link']}>
               Contact
             </Link>
           </li>
```

Comment:
It seems like you can import the entire menu. It would be very smart to implement a map function to show all elements automatically.